Updated the JavaScript files code from CommonJS to EC Modules

AyzeLYC opened the pull request:
Updated all the :

const var_name = require('library_name');
const ineer_namespace_name = require('library_name').inneer_namespace;

to :

const var_name = await import("library_name");
const {ineer_namespace_name} = await import("library_name");

If NodeJS itself has the required libraries then i used node:library_name to not have to download the libraries through npm and save bandwith and time

AyzeLYC committed:
Updated test

AyzeLYC committed:
Updated test

AyzeLYC committed:
Update start

AyzeLYC committed:
Updated test

AyzeLYC committed:
Update publish

AyzeLYC committed:
Updated index.js

AyzeLYC committed:
Updated start

AyzeLYC committed:
Updated test

AyzeLYC committed:
Updated publish

AyzeLYC committed:
Updated index.js

piranna commented:
Please continúe working on that, this is the kind of updates i would like to keep seeing on NodeOS :-) Have you been able to update the Node.ja version too?

AyzeLYC commented:
> Please continúe working on that, this is the kind of updates i would like to keep seeing on NodeOS :-) Have you been able to update the Node.ja version too?

right now i'm working on a personal project but i've been able to modify some of the code, i'll compile everything and run it using a virtual machine to see if everything works properly

AyzeLYC committed:
Updated install-dependencies

AyzeLYC committed:
Updated index.js

AyzeLYC committed:
Updated README.md

AyzeLYC commented:
everything is up to date and should be working properly, once i'll have tested everything and made sure than it works i'll confirm than it works here

AyzeLYC committed:
Updated start

AyzeLYC committed:
Updated start

AyzeLYC committed:
Updated publish

AyzeLYC committed:
Updated start

AyzeLYC committed:
Updated test

AyzeLYC committed:
Updated postinstall

AyzeLYC committed:
Updated build

AyzeLYC committed:
Updated BigRedButton

AyzeLYC committed:
Updated dockerBuild

AyzeLYC committed:
Updated postbuild

AyzeLYC committed:
Updated unbuild

AyzeLYC committed:
Updated install-dependencies

AyzeLYC committed:
Updated installUSB

AyzeLYC committed:
Updated Dockerfile

AyzeLYC committed:
Updated Dockerfile-initramfs

AyzeLYC committed:
Updated vagga.yaml

AyzeLYC committed:
Updated disk.sh

AyzeLYC committed:
Updated docker.sh

AyzeLYC committed:
Updated img.sh

AyzeLYC committed:
Updated iso.sh

AyzeLYC committed:
Updated qemu.sh

AyzeLYC committed:
Updated runner_header.sh

AyzeLYC committed:
Updated index.js

AyzeLYC committed:
Updated index.js

AyzeLYC committed:
Updated LICENSE.txt

AyzeLYC commented:
i modified all i've done to make sure than there's absolutely no bug when it's running and made it so that rather than using multiple variables to call all the different functions of the dependancies it's instead included in an array called dependancies

i also modified all the .sh files so that the license is up to date

AyzeLYC committed:
Updated index.js

AyzeLYC committed:
Updated publish

AyzeLYC committed:
Updated start

AyzeLYC committed:
Updated test

AyzeLYC committed:
Updated publish

AyzeLYC committed:
Updated test

AyzeLYC commented:
now everything is good, i just need to compile everything and test it on a VM and i'll see if it works properly or not

AyzeLYC commented:
well i have to update the dependancies because some of them are totally out of date and it's causing npm to return errors, it's going to be fun
